vlo-importer/src/main/java/eu/clarin/cmdi/vlo/importer/CMDIData.java

```diff
@@ -152,6 +154,20 @@ private void handleDocField(String name, String value, boolean caseInsensitive)
         return searchPageResources;
     }
 
+    /**
```

Comment:
Since we only do accomodate for a `.isEmpty()` check in `MetadataImporter` we could also/instead have a method `public boolean hasResources()` that returns the logical conjunction of all lists being empty? My gut feeling says that this would be more efficient as it does not involve object creation or iteration over collections.

Alternatively (for something as versatile as the current solution) we can use `Iterables.concat` (from Guava, already available) or `Streams.concat` and operate on that (i.e. `Iterables.isEmpty(iteratable)` or `stream.findAny().isPresent()`).

Reply:
I also thought about using Guava or the new Stream API, but I normally prefer the "verbose" implementation, that is often easier to read. I see your point with the Boolean return type and will change the implementation accordingly.

Reply:
I tend to agree with the verbosity/readability aspect. Using streams can make things harder to interpret. The Iterator utility methods provide a nice compromise IMO. But the more specific boolean method would also be my choice in this case.